java/src/jmri/jmrit/operations/setup/JmritOperationsSetupBundle.properties

```diff
@@ -74,7 +74,8 @@ AddLogo             = Add Logo
 RemoveLogo          = Remove Logo
 Reset               = Reset
 
-#The following keys must exactly match their English translation, used as tags in the xml file.
+#The following keys must not be modified or backwards compatibility will be broken, used as tags in the xml file.
```

Comment:
It's okay to translate, that's not the issue.  Modifying the English version could cause a user setting to be lost after upgrading.  The user selections are saved in English, not in the selected language.